app/src/main/res/values-ar/strings.xml

```diff
@@ -173,7 +173,6 @@
     <string name="nc_new_password">كلمة سرية جديدة</string>
     <string name="nc_nick_guest">ضيف</string>
     <string name="nc_no">لا</string>
-    <string name="nc_no_messages_yet">ليس لديك رسائل بعد</string>
```

Comment:
I think you only need to remove the english string. The rest should be done by transifex and the sync bot.

Reply:
Oops, didn't know that. Just fixed it.